modules/get-started/pages/whats-new.adoc

```diff
@@ -5,78 +5,17 @@
 This topic includes new content added in version {page-component-version}. For a complete list of all product updates, see the https://github.com/redpanda-data/redpanda/releases/[Redpanda release notes^]. See also:
 
 * xref:redpanda-cloud:get-started:whats-new-cloud.adoc[] 
-* xref:redpanda-cloud:get-started:cloud-overview.adoc#redpanda-cloud-vs-self-hosted-feature-compatibility[Redpanda Cloud vs self-hosted feature compatibility]
+* xref:redpanda-cloud:get-started:cloud-overview.adoc#redpanda-cloud-vs-self-managed-feature-compatibility[Redpanda Cloud vs Self-Managed feature compatibility]
 
-== FIPS compliance
 
-Redpanda now provides a FIPS 140-compliant binary to support running Redpanda in environments requiring FIPS-compliant software and to meet certain requirements for organizations undergoing certification through government compliance programs, such as FedRAMP. When using the FIPS binary, both Redpanda and `rpk` leverage FIPS-compliant OpenSSL libraries and ciphers for all encryption activity inside Redpanda, including encryption of data in transit. Install the `redpanda-fips` package to xref:manage:security/fips-compliance.adoc[run Redpanda in FIPS-compliance mode].
+== Mountable topics
 
-== Schema Registry support for JSON
+For topics with Tiered Storage enabled, you can xref:manage:mountable-topics.adoc[unmount a topic] to detach segment data that is still on disk to object storage, and you can mount a topic from object storage to attach the topic data to either the same origin cluster or a different one. This allows you to hibernate a topic and free up and reclaim system resources taken up by the topic, or you can migrate a topic to a different cluster.
```

Comment:
```suggestion
For topics with Tiered Storage enabled, you can unmount a topic to detach segment data that is still on disk to object storage, and mount that topic to either the same origin cluster, or a different one. This allows you to hibernate a topic and free up system resources taken up by the topic, or migrate a topic to a different cluster. See xref:manage:mountable-topics.adoc[Mountable topics] for details.
```
Feel free to copy edit, I'm also happy to just link to xref:manage:mountable-topics.adoc[unmount a topic] like you've done, and skip the additional "See Mountable topics" sentence.

Reply:
thanks @kbatuigas